python/tank/bootstrap/resolver.py

```diff
@@ -671,40 +708,23 @@ def resolve_shotgun_configuration(
             # 1 windows/linux/mac path
             # 2 descriptor
             # 3 sg_descriptor
-
-            path = ShotgunPath.from_shotgun_dict(pipeline_config)
-
-            if path and not path.current_os:
+            if pipeline_config["config_descriptor"] is None:
                 log.debug(
-                    "No path set for %s on the Pipeline Configuration \"%s\" (id %d).",
+                    "No location set for %s on the Pipeline Configuration \"%s\" (id %d).",
```

Comment:
I don't to say descriptor because it can be paths, uploaded config or descriptor field.